structure/0006-governance.md

```diff
@@ -2,53 +2,152 @@
 
-#### Rationale and Alternatives
-Can be omitted. Who is currently responsible for existing projects this team will takeover? What teams might become responsible for new projects proposed in this document if the team itself is deemed unnecessary?
+At the start of the voting process, each organization members may nominate themselves. Nominations must be submitted to the EOB within one week of the start of the process, after which time the vote is opened.
 
-#### Prior Art
-Can usually be omitted. Is only necessary if there are prior organizations/projects with very similar goals.
+After one week of preparation and nomination submission, nominations must close, the list of candidates must be published, and the vote must begin, lasting one week. The list of voters cannot change beyond that point, unless a voter is removed from the organization adn pauses serious threat to the vote integrity. The system used to conduct the vote must allow voters to change their votes, and enable voters to rank the candidates in order of preference, while also having the option to not rank any given candidate. All organization members (which at this time is anyone with the "Community Team" or "Quilt Developer" role in Discord) are able to vote. During the vote, voters can contact the EOB about any concerns they have over the vote (for example, concerns about a candidate being elected, or concerns about the vote being influenced by an external force), to be dealt with as outlined in the paragraph below.
+
+If the EOB receives any concerns over a candidate, either internally or submitted by voter, they must decide if the concerns are worth investigating. If they are not, no further action is taken. If they are, the team must investigate, and, if significant results are found, must publish the results to the voters, so that they can factor them into their decision. The EOB cannot prevent any candidate from running, this is to ensure that they don't have direct control over who can be elected. If, however, the vote is due to end before the EOB can complete their investigation, they may delay the end of the vote by up to 30 additional days.
+
+After the voting period concludes, the EOB may take up to one week to count the votes. In the event of a tie, votes by existing admin board members will count as three votes. The EOB may privately inform the winning candidate, before publishing the results to the organization members and subsequently the community at large.
```

Comment:
We do not have a way for these votes to be counted in triplicate because the services we use that handle the voting process make those votes anonymous.

I believe this is an important part of the existing procedure, so this probably needs to be changed.

Reply:
maybe just having the existing AB vote on the tied candidates would be a better solution?